launch/launch/actions/execute_process.py

```diff
@@ -216,12 +216,12 @@ def __init__(
         self.__log_cmd = log_cmd
         self.__on_exit = on_exit
 
-        self.__process_event_args = None  # type: Optional[Dict[Text, Any]]
-        self._subprocess_protocol = None  # type: Optional[Any]
+        self.__process_event_args = None   # type: Optional[Dict[Text, Any]]
```

Comment:
Is this space-related change necessary or can it be reverted (same for other instances)?

Reply:
Ah no these were the result of an overzealous sed. I'll fix that up.